lib/plug_image_processing/config.ex

```diff
@@ -24,19 +24,24 @@ defmodule PlugImageProcessing.Config do
     Middlewares.CacheHeaders
   ]
 
+  @enforce_keys ~w(path)a
   defstruct path: nil,
             sources: @sources,
             operations: @operations,
             middlewares: @middlewares,
```

Comment:
@simonprev I don’t think you can pluralize "middleware" by appending an "s". It’s like software — both singular and prural forms are "software" 🤓

Reply:
TIL